src/lib/cpp/files.cc

```diff
@@ -13,9 +13,28 @@ You should have received a copy of the GNU General Public License along with thi
 #include <sys/stat.h>
 #include <sstream>
 
+/**
+ * \file
+ * \brief Utility functions for file handling
+ */
+
+/**
+ * \namespace fo
+ * \brief `fo` namespace holds the FOSSology library functions.
+ */
 namespace fo
 {
 
+  /**
+   * \brief Reads the content of a file and return it as a string.
+   *
+   * Read the content of the file defined by the filename. Function also limits
+   * the length of the file content by using maximumBytes.
+   * \param filename     Path of the file to read.
+   * \param maximumBytes Maximum length to read (set -1 to read full length).
+   * \return The file content limited by maximumBytes as string.
+   * \todo respect limit of maximumBytes
```

Comment:
let's have all the same issue for code todos (if that todo is serious still today)